docs/docs/user/references/cli-commands.md

```diff
@@ -126,6 +127,26 @@ $ kpops generate [OPTIONS] PIPELINE_PATH
 * `--verbose / --no-verbose`: Enable verbose printing  [default: no-verbose]
 * `--help`: Show this message and exit.
 
+## `kpops init`
+
+Initialize a new a KPOps project.
+
+**Usage**:
+
+```console
+$ kpops init [OPTIONS] PATH
+```
+
+**Arguments**:
+
+* `PATH`: Path for a new KPOps project.  [required]
+
+**Options**:
+
+* `--name TEXT`: Name of the new KPOps project. A new directory with the provided name will be created. Leave empty to use the existing dir provided via `[PATH]`.
```

Comment:
is this option necessary?

I'd say we can just create the provided `PATH` if it doesn't exist

Reply:
Good point, it isn't.